hxrsnd/sndsystem.py

```diff
@@ -103,21 +97,18 @@ class SplitAndDelay(SndDevice):
     E2 = Cmp(Energy2Macro, "", desc="CC Energy")
     delay = Cmp(DelayMacro, "", desc="Delay")
 
-    # DAQ
-    daq = Cmp(Daq, None, platform=1)
-    
-    def __init__(self, prefix, name=None, *args, **kwargs):
+    def __init__(self, prefix, name=None, daq=None, RE=None,
+                 *args, **kwargs):
         super().__init__(prefix, name=name, *args, **kwargs)
+        self.daq = daq
+        self.RE = RE
```

Comment:
The plan is to pass these in the `hutch_python` beamline.py file?

Reply:
Yes. See https://github.com/pcdshub/xcs/blob/73b2fe065e808e845b622b4e25589abdc796cb80/xcs/beamline.py#L4-L7